parser.y

```diff
@@ -11094,8 +11094,27 @@ CreateBindingStmt:
 		hintedSelStmt.SetText(strings.TrimSpace(parser.src[startOffset:]))
 
 		x := &ast.CreateBindingStmt{
-			OriginSel:   selStmt,
-			HintedSel:   hintedSelStmt,
+			OriginNode:  selStmt,
+			HintedNode:  hintedSelStmt,
+			GlobalScope: $2.(bool),
+		}
+
+		$$ = x
+	}
+|	"CREATE" GlobalScope "BINDING" "FOR" SetOprStmt "USING" SetOprStmt
```

Comment:
`SetOprStmt` don't full contain `SetOprStmt`. Could you parse SQL like:
```sql
create binding for select 1 union select 2 intersect select 3 using select 1 union select 2 intersect select 3
```
Please use `SetOprStmt1` instead it and remove `SelectStmt`.

Reply:
And also please add some parser tests for binding and set operator statements.